schema/project-level/project-package-schema.json

```diff
@@ -24,8 +24,14 @@
       "type": "string",
       "format": "date-time"
     },
+    "version": {
+      "title": "OC4IDS schema version",
+      "description": "The version of the OC4IDS schema used in this package, expressed as major.minor[-tag]. For example: 1.0-beta or 1.1",
```

Comment:
@jpmckinney @kindly are you happy with this approach for declaring the version in the package metadata?

Reply:
@duncandewhurst 

I would prefer just major.minor and miss off the tag bit.  This is what it is in core OCDS and I would copy the regex from there too.

I do not think you want data in the wild to differentiate between "patch" releases as they really should be compatible with each other.  For a beta I would call it something like 0.9 if you are expecting any major schema changes and 1.0 if not (so hopefully people who are developing against the beta will not have to change once its released)

Reply:
I agree with @kindly